docs/commands/data/display.rst

```diff
@@ -0,0 +1,24 @@
+.. index:: display
+.. _display:
+
+Set a Display Expression (`display`)
+------------------------------------
+
+**display** [ *format* ] *expression*
```

Comment:
Is bashdb really supporting `format`? According to `help display` it's not supporting it.